### ISSUE TITLE
Dart VM Service web page still uses Observatory branding

### ISSUE THREAD

**Issue by mit-mit** [dart-lang/sdk#47390]
When running a Flutter app, a message like this is printed:

```
Connecting to VM Service at http://127.0.0.1:49955/2P6qxL3ZOIU=/
```
![Screen Shot 2021-10-06 at 14 09 18](https://user-images.githubusercontent.com/13644170/136199426-b4e5b6c7-2f74-4125-b4f2-9d21221f6a69.png)

If you open this in Chrome, you get a web page that's called "Observatory"

Given we don't really use Observatory as a developer tool anymore, I suggest we change this branding (both tab name and the blue banner) to simply say "Dart VM Service".

**Comment by mit-mit** [comment]
cc @bkonyi

**Comment by bkonyi** [comment]
I don't think this makes sense to do at this point given that the VM still prints `Observatory listening on http://127.0.0.1:...`. We should change this string before dropping the Observatory name, but that's a rather large breaking change that will require a lot of coordination to roll fixes into Flutter and google3. There's a breaking change request (still pending AFAIK) that tracks that work: https://github.com/dart-lang/sdk/issues/46756.
